### PR TITLE
Adds deprecation notice

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Adds the deprecation notice in a warning admonition for now, until we have a mechanism for a banner.

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Fixes https://github.com/timescale/docs-private/issues/19

### CODE REVIEW

File: mst/mst-multi-node.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
-<highlight type="important">
-Timescale currently offers two hosting options. If you are a Timescale Cloud
-(previously called Timescale Forge) user, please use the documentation for
-[setting up multi-node on Cloud](/cloud/latest/cloud-multi-node) instead.
+# Setting up TimescaleDB 2.0 multi-node on Managed Service for TimescaleDB
+
+<highlight type="warning">
+This section describes a feature that is deprecated on Managed Service for
+TimescaleDB. We strongly recommend that you do not use this feature in a
+production environment. If you need more information, please
```
Comment (jfjoly) [inline mst/mst-multi-node.md:6]: we could suggest users to use TS Cloud for multi-node.
"If you need multi-node, you may check [Timescale Cloud](https://docs.timescale.com/timescaledb/latest/overview/deployment-options/#managed-service-for-timescaledb), and for more information, please contact the support team."

**Review by jfjoly — APPROVED** [review]
Looks good to me. I made a suggestion but happy to merge as it is now.

**Review by charislam — APPROVED** [review]

File: mst/mst-multi-node.md [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
-Timescale currently offers two hosting options. If you are a Timescale Cloud
-(previously called Timescale Forge) user, please use the documentation for
-[setting up multi-node on Cloud](/cloud/latest/cloud-multi-node) instead.
+# Setting up TimescaleDB 2.0 multi-node on Managed Service for TimescaleDB
+
+<highlight type="warning">
+This section describes a feature that is deprecated on Managed Service for
+TimescaleDB. We strongly recommend that you do not use this feature in a
+production environment. If you need more information, please  
+[contact the support team](https://www.timescale.com/support).
```
Comment (Loquacity) [inline mst/mst-multi-node.md:7]:
```suggestion
This section describes a feature that is deprecated on Managed Service for
TimescaleDB. We strongly recommend that you do not use this feature in a
production environment. This feature is, however, available on Timescale 
Cloud. If you need more information, please  
[contact the support team](https://www.timescale.com/support).
```

**Review by Rajakavitha1 — APPROVED** [review]

File: mst/mst-multi-node.md [hunk 3]
```diff
@@ -1,10 +1,12 @@
-<highlight type="important">
-Timescale currently offers two hosting options. If you are a Timescale Cloud
-(previously called Timescale Forge) user, please use the documentation for
-[setting up multi-node on Cloud](/cloud/latest/cloud-multi-node) instead.
+# Setting up TimescaleDB 2.0 multi-node on Managed Service for TimescaleDB
+
+<highlight type="warning">
```
Comment (jacobprall) [inline mst/mst-multi-node.md:3]:
```suggestion
<highlight type="deprecation">
```

**Comment by Loquacity** [comment]
image

Looks great! Thanks for getting this done 😁

**Pull request merged**